### ISSUE TITLE
Keyboard focus doesn't move into Dialog when it is opened in IE

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#208]

**Comment by jessegreenberg** [comment]
This is related to #206 - the fix in that issue works for elements in the PDOM since we have complete control over those - but IE is trying to focus other things in the document that are outside of the PDOM (like scripts, divs, and svg elements since all of these have tabindex="0" in IE).

**Comment by jessegreenberg** [comment]
```getNextPreviousFocusable``` of AccessibilityUtil could be limited to the parallel DOM.  But then it would need a reference to the sim display

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/john-travoltage/issues/208#issuecomment-291217131

No good, AccessibilityUtil in scenery should be for general DOM traversal and manipulation, and can't be dependent on phet or sim.

**Comment by jessegreenberg** [comment]
The callback to open the menu could limit the search to the PDOM.  Display needs a getter for the root accessible DOM element.

**Comment by jessegreenberg** [comment]
Or Dialog could handle focus of the first element?

**Comment by jessegreenberg** [comment]
But we don't want focus if dialog is opened with mouse, so Dialog would need knowledge of how it was opened to determine whether or not first focusable element should receive focus.

https://github.com/phetsims/john-travoltage/issues/208#issuecomment-291217834 Seems best at the moment.

**Comment by jessegreenberg** [comment]
This is fixed with the above commits, closing.
